docs/source/performance.mdx

```diff
@@ -15,10 +15,466 @@ limitations under the License.
 -->
 
 # Performance and Scalability: How To Fit a Bigger Model and Train It Faster
+> _Or how to escape the dreaded "RuntimeError: CUDA error: out of memory" error._
```

Comment:
A new line here would be nice for readability.

Reply:
added